docs/getting-started.mdx

```diff
@@ -8,46 +8,59 @@ import { AssetsProvider } from '@site/src/components/assets-context';
 import DownloadCard from '@site/src/components/download-card';
 import DownloadCardStyles from '@site/src/css/download-card.module.css';
 
-
-## Get an Account
-
-<Columns className='text--center getting-started-cards'> 
-  <Column style={{paddingBottom: 20}}>
-    <Card shadow='tl'>
-      <CardHeader>
-        <h3>Hosted zrokNet</h3>
-      </CardHeader>
-      <CardBody>
-        Use NetFoundry's public zrok instance.
-      </CardBody>
-      <CardFooter>
-        <a href="https://myzrok.io/">
-        <button className='button button--secondary button--block'>Get an Account</button>
-        </a>
-      </CardFooter>
-    </Card>
-  </Column>
-  <Column style={{paddingBottom: 20}}>
-    <Card shadow='tl'>
-      <CardHeader>
-        <h3>Self-Hosted zrok</h3>
-      </CardHeader>
-      <CardBody>
-        Run a zrok instance on Linux, Docker, or Kubernetes.
-      </CardBody>
-      <CardFooter>
-        <a href="/docs/category/self-hosting/">
-        <button className='button button--secondary button--block'>Guides</button>
-        </a>
-      </CardFooter>
-    </Card>
-  </Column>
+## Secure resource sharing made easy
+
+<Columns className='text--center getting-started-cards'>
+    <Column style={{paddingBottom: 20}}>
+        <Card shadow='tl'>
+            <CardHeader>
+                <h3>Hosted zrok</h3>
+            </CardHeader>
+            <CardBody>
+                Use NetFoundry's public zrok instance.
+            </CardBody>
+            <CardFooter>
+                <a href="https://myzrok.io/">
+                    <button className='button button--secondary button--block'>Get an Account</button>
+                </a>
+            </CardFooter>
+        </Card>
+    </Column>
+    <Column style={{paddingBottom: 20}}>
+        <Card shadow='tl'>
+            <CardHeader>
+                <h3>Self-Hosted zrok</h3>
+            </CardHeader>
+            <CardBody>
+                Run a zrok instance on Linux, Docker, or Kubernetes.
+            </CardBody>
+            <CardFooter>
+                <a href="/docs/category/self-hosting/">
+                    <button className='button button--secondary button--block'>Guides</button>
+                </a>
+            </CardFooter>
+        </Card>
+    </Column>
 </Columns>
 
+* Sign up for a hosted zrok account in the link above to enable your account token
```

Comment:
```suggestion
* Sign up for a hosted zrok account in the link above to get your account token
```
"Enable your account token" is slightly misleading because they only need to obtain it, and we usually say "enable an environment" (with an account token).